app/assets/javascripts/app/reservation_time_checker.coffee

```diff
@@ -1,8 +1,12 @@
 class window.ReservationTimeChecker
   constructor: (@selector, @alertId = 'duration-alert')->
-    @initAlert()
-    @respondToChange()
+    if @validPage()
+      @initAlert()
+      @respondToChange()
 
+  validPage: ->
+    # These variable are not set on the admin reservation pages
+    reserveInterval? && reserveMinimum? && reserveMaximum?
```

Comment:
If I wanted to call it `adminReservationPage`, I would need to invert the logic to be `!reserveInterval? || !reserveMinimum? || !reserveMaximum?` which just seems gross. We want to run the code on the regular user-facing reservation page, and something like `regularReservationPage` seemed weird as well.